src/static/kit/scripts.js

```diff
@@ -826,6 +826,10 @@ function default_tabmenu_attacher(x) {
 	x.on('tabmenuTabHasChanged', function() { x.nextAll().remove(); });
 	x.appendTo(this);
 }
+function tabmenu_attacher_with_change_callback(tabmenu_changed_callback, x) {
+	x.in('tabmenuTabHasChanged', tabmenu_changed_callback);
```

Comment:
```suggestion
	x.on('tabmenuTabHasChanged', tabmenu_changed_callback);
```
I think that is better